pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -307,7 +308,9 @@ func getNetwork(ctx context.Context, path string, finder *find.Finder, client *v
 
 	var networkChoices []string
 	for _, network := range networks {
-		if network.Reference().Type == distributedVirtualPortGroupType || network.Reference().Type == networkType {
+		if network.Reference().Type == distributedVirtualPortGroupType ||
```

Comment:
at this point should we just move it to In semantics.
types:= sets.NewString()
types.Has(network.Reference().Type)